src/client/tile.cpp

```diff
@@ -1921,7 +1938,7 @@ static void apply_colorize(video::IImage *dst, v2u32 dst_pos, v2u32 size,
 		for (u32 x = dst_pos.X; x < dst_pos.X + size.X; x++) {
 			dst_c = dst->getPixel(x, y);
 			if (dst_c.getAlpha() > 0) {
-				dst_c = color.getInterpolated(dst_c, interp);
+				dst_c = blit_pixel(color, dst_c, interp);
```

Comment:
this is incorrect, blit_pixel will divide interp by / 255 another time

Reply:
I ended up backing out the getInterpolated -> blitPixel change in the `colorize` and the dead-code `blit_with_interpolate_overlay` areas, since it looks like the documentation for them suggests that getInterpolated was actually the more correct behavior.

I'm actually a little confused about the `blit_with_alpha_overlay` behavior.  It looks like it's only used to draw the crack texture, and it only applies the texture to pixels with alpha = 255.  This makes the behavior obviously correct for fully transparent or fully opaque pixels, but what it should do for middle-alpha pixels is ill-defined.